frontend/packages/dev-console/src/components/health-checks/AddHealthCheckPage.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import { RouteComponentProps } from 'react-router-dom';
+import { PageHeading, FirehoseResource, Firehose } from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { ServiceModel as KnativeServiceModel } from '@console/knative-plugin';
+import AddHealthCheckWrapper from './AddHealthCheckWrapper';
+
+type AddHealthCheckProps = RouteComponentProps<{
+  ns: string;
+  kind: string;
+  name: string;
+  containerName: string;
+}>;
+
+const AddHealthCheckPage: React.FC<AddHealthCheckProps> = ({ match }) => {
+  const { ns, kind, name, containerName } = match.params;
+  const resource: FirehoseResource[] = [
+    {
+      kind: kind === KnativeServiceModel.kind ? referenceForModel(KnativeServiceModel) : kind,
+      namespace: ns,
+      isList: false,
+      name,
+      prop: 'resource',
+      optional: true,
+    },
+  ];
+
+  return (
+    <>
+      <Helmet>
+        <title>Health Checks</title>
+      </Helmet>
+      <div>
+        <PageHeading title="Health Checks" />
```

Comment:
When adding the page should be named `Add Health Checks` and when editing it is `Edit Health Checks`